### ISSUE TITLE
Training loss did not go down

### ISSUE THREAD

**Issue by yxsysu** [baofff/U-ViT#13]
【Setting:】 I modified the config in imagenet256_uvit_huge.py to train the U-ViT huge in another dataset and utilized the imagenet256 pretrained checkpoint for initialization. Except for the setting related to the dataset,  I keep the other settings such as batch size (1024) unchanged. 
【Problem?】During the first 16K iterations, I found the training loss fluctuated within a certain range: [0.15, 0.18] and did not go down.
The generated samples look like samples from the new domain, although some artifacts still exist.
I am curious about why the loss did not go down. 
Is this a normal training phenomenon?
Thank you for your excellent work.

**Comment by nieshenx** [comment]
When training a diffusion model, during the early stages of training, such as 30k iterations for uvit_huge in Imagenet 256 datasets, the training loss will be observed to decrease rapidly. In the middle and later stages of training, even with the use of tools like wandb or TensorBoard, it is almost impossible to visually observe the decrease in training loss. If you set the "smoothing" of the train_loss vs steps curve (provided by tools like wandb or TensorBoard) to the maximum, you can see that the training loss is still slowly decreasing. 
If you want to use pretrained uvit in a new domain, perhaps visually inspecting the quality of the sampled images, or using quantitative metrics like FID, would be a better way to monitor the experiment. Of course, this is just a suggestion.

**Comment by yxsysu** [comment]
Thank! @NieShenRuc
